### PR TITLE
add accessibility section for `Alert`

### LINKED ISSUE #1578: Add accessibility section for `alert`
### Subject

accessibility section for `alert`

#### Create an accessibility section referring to the links below
https://yamada-ui.com/components/disclosure/tabs/accessibility#aria-roles-and-attributes ( https://github.com/yamada-ui/yamada-ui/issues/1552 )
https://mantine.dev/core/alert/#accessibility
https://mui.com/material-ui/react-alert/#accessibility

#### Include the following content

- link for W3C
  - https://www.w3.org/WAI/ARIA/apg/patterns/alert/
  - https://www.w3.org/WAI/ARIA/apg/patterns/alert/examples/alert/
### Are you willing to participate in creating this documentation page and create a pull request?

None

### PR DESCRIPTION
**Pull request by setodeve**
Closes #1578 

## Description
Added accessibility documentation for `Alert`.

## Is this a breaking change (Yes/No):
No

### CODE REVIEW

**Review by kurakke — DISMISSED** [review]

**Review by cidkumagai — APPROVED** [review]
LGTM!

**Comment by hirotomoyamada** [comment]
@setodeve @cidkumagai @kurakke 

I reviewed only the Japanese documentation page, but the English documentation page should be corrected as well.

File: docs/contents/components/feedback/alert/accessibility.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+## Keyboard Navigation
+
+No keyboard interaction needed.
+
+## ARIA Roles and Attributes
+
+| Component | Roles and Attributes    | Usage                                                                               |
+| --------- | ----------------------- | ----------------------------------------------------------------------------------- |
+| `Alert`   | `	role="alert"`          | Indicates that it is a alert.                                                       |
+|           | `	aria-live="assertive"` | Indicates that it is a immidiate notification.                                      |
```
Comment (kurakke) [inline docs/contents/components/feedback/alert/accessibility.mdx:20]: Please change "a" to "an" before words that start with a vowel sound. For example, "a alert" should be "an alert."

File: docs/contents/components/feedback/alert/accessibility.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+|           | `	aria-atomic=true`      | Indicates that it is the entire content of the alert elemennt as the alert message. |
```
Comment (kurakke) [inline docs/contents/components/feedback/alert/accessibility.mdx:21]: Please change "elemennt" to "element" to correct the spelling error.

File: docs/contents/components/feedback/alert/accessibility.mdx (hunk 1 above)
Comment (kurakke) [inline docs/contents/components/feedback/alert/accessibility.mdx:20]: Please correct the spelling error: change "immidiate" to "immediate."

**Comment by setodeve** [comment]
@kurakke 
Sorry, I have corrected the points you mentioned. 
Could you please review it again?

**Review by kurakke — DISMISSED** [review]
> @kurakke Sorry, I have corrected the points you mentioned. Could you please review it again?

@setodeve 
Thank you!
LGTM!!!

**Review by fffk10 — APPROVED** [review]

File: docs/contents/components/feedback/alert/accessibility.ja.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+## キーボード操作
+
+キーボード操作は必要ありません。
+
+## ARIAロールと属性
+
+| コンポーネント | ロールと属性            | 使用方法                                                           |
+| -------------- | ----------------------- | ------------------------------------------------------------------ |
+| `Alert`        | `role="alert"`          | これはアラートであることを示します。                               |
+|                | `aria-live="assertive"` | これは即時通知であることを示します。                               |
```
Comment (hirotomoyamada) [inline docs/contents/components/feedback/alert/accessibility.ja.mdx:20]: The `aria-live="assertive"` is included in `role="alert"`. Delete this line.

File: docs/contents/components/feedback/alert/accessibility.ja.mdx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+|                | `aria-atomic=true`      | アラート要素の全体の内容がアラートメッセージであることを示します。 |
```
Comment (hirotomoyamada) [inline docs/contents/components/feedback/alert/accessibility.ja.mdx:21]: The `aria-atomic="true"` is included in `role="alert"`. Delete this line.

File: docs/contents/components/feedback/alert/accessibility.mdx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+No keyboard interaction needed.
+
+## ARIA Roles and Attributes
+
+| Component | Roles and Attributes    | Usage                                                                              |
+| --------- | ----------------------- | ---------------------------------------------------------------------------------- |
+| `Alert`   | `role="alert"`          | Indicates that it is an alert.                                                     |
+|           | `aria-live="assertive"` | Indicates that it is an immediate notification.                                    |
+|           | `aria-atomic=true`      | Indicates that it is the entire content of the alert element as the alert message. |
```
Comment (hirotomoyamada) [inline docs/contents/components/feedback/alert/accessibility.mdx:21]: The `aria-atomic="true"` is included in `role="alert"`. Delete this line.

File: docs/contents/components/feedback/alert/accessibility.mdx [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+## Keyboard Navigation
+
+No keyboard interaction needed.
+
+## ARIA Roles and Attributes
+
+| Component | Roles and Attributes    | Usage                                                                              |
+| --------- | ----------------------- | ---------------------------------------------------------------------------------- |
+| `Alert`   | `role="alert"`          | Indicates that it is an alert.                                                     |
+|           | `aria-live="assertive"` | Indicates that it is an immediate notification.                                    |
```
Comment (hirotomoyamada) [inline docs/contents/components/feedback/alert/accessibility.mdx:20]: The `aria-live="assertive"` is included in `role="alert"`. Delete this line.

File: docs/contents/components/feedback/alert/accessibility.mdx [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+title: Alert
+description: "`Alert`is a component that conveys information to the user."
+package_name: "@yamada-ui/alert"
+with_description: true
+tab: Accessibility
+---
+
+`Alert` follows the [WAI-ARIA - Alert Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/alert/)for accessibility.
+
+## Keyboard Navigation
```
Comment (hirotomoyamada) [inline docs/contents/components/feedback/alert/accessibility.mdx:11]: If there is no keyboard operation, delete the entire section.

**Comment by setodeve** [comment]
@hirotomoyamada 

I have corrected the points you mentioned.
Could you please review it again?

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**